tests/resources/Harbor-Util.robot

```diff
@@ -20,48 +20,12 @@ Library  OperatingSystem
 *** Variables ***
 ${HARBOR_SHORT_VERSION}  0.5.0
 ${HARBOR_VERSION}  harbor_0.5.0-9e4c90e
-${MY_PROJECTS_TABLE}  div.table-body-container > table.table.table-pane
+${MY_PROJECTS_TABLE}  harbor-app > harbor-shell > clr-main-container > div > div > project > div > div > list-project
+${MY_MEMBERS_TABLE}  body > harbor-app > harbor-shell > clr-main-container > div > div > project-detail > ng-component > div > div:nth-child(2) > clr-datagrid > div > div > div > div.datagrid-body
 ${harbor_cert}  getcert
+${ova_harbor_admin_password}  harbor-admin-passwd
 
 *** Keywords ***
-Install Harbor To Test Server
```

Comment:
Even though this uses the outdated installer it needs to be kept for now since there are other tests that use it